lisa/features/gpu.py

```diff
@@ -135,23 +139,30 @@ def check_support(self) -> None:
         if not self._is_supported():
             raise SkippedException(f"GPU is not supported with distro {self._node.os}")
 
-    def install_compute_sdk(
-        self, driver: ComputeSDK = ComputeSDK.CUDA, version: str = ""
-    ) -> None:
+    def install_compute_sdk(self, version: str = "") -> str:
+        gpu_vendor = ""
+
         # install GPU dependencies before installing driver
         self._install_gpu_dep()
 
+        if isinstance(self._node.os, Redhat):
+            # install LIS driver if not already installed.
+            lisdriver = self._node.tools[LisDriver]
+            if not lisdriver.get_version():
```

Comment:
I thought Tools.__get_item__() would take care of this, no?

Reply:
you are right. will update